### PR TITLE
Update tutorial with v1beta1

### PR DESCRIPTION
**Pull request by savitaashture**
JIRA: [SRVKP-650](https://issues.redhat.com/browse/SRVKP-650)

### CODE REVIEW

**Comment by pthangad** [comment]
@savitaashture You need to change trigger template [pipelineRun](https://github.com/savitaashture/pipelines-tutorial/blob/updatetutorial/03_triggers/02_template.yaml#L38) resource from v1alpha1 to v1beta1

File: 01_pipeline/01_apply_manifest_task.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
-apiVersion: tekton.dev/v1alpha1
+apiVersion: tekton.dev/v1beta1
```
Comment (nikhil-thomas) [inline 01_pipeline/01_apply_manifest_task.yaml:1]: could you make it `triggers.tekton.dev`

Comment (nikhil-thomas) [reply 01_pipeline/01_apply_manifest_task.yaml:1]: and it all the other Triggers Resources

Comment (nikhil-thomas) [reply 01_pipeline/01_apply_manifest_task.yaml:1]: please ignore this :D

Comment (savitaashture) [reply 01_pipeline/01_apply_manifest_task.yaml:1]: I think `triggers.tekton.dev` only for trigger resources not for pipeline resources right

**Review by vdemeester — APPROVED** [review]
/lgtm

**Pull request merged**